django/db/migrations/recorder.py

```diff
@@ -71,6 +75,7 @@ def record_applied(self, app, name):
         """
         self.ensure_schema()
         self.migration_qs.create(app=app, name=name)
+        logger.info('Marked migration %s.%s as applied' % (app, name))
```

Comment:
So, this message and the "applied..." message come from the same logger at the same severity, and cannot be "separated"; I suspect this should be at `debug` level.